projects/gis/src/app/main/main.component.html

```diff
@@ -3,6 +3,16 @@
 
     <geoengine-layer-list-menu></geoengine-layer-list-menu>
 
+    <ng-container *ngIf="config.BRANDING.HOMEPAGE as homepage">
+        <div class="divider"></div>
+
+        <a [href]="homepage.URL" mat-flat-button color="primary" class="homepage-button">
+            <mat-icon>keyboard_return</mat-icon>
+            <span></span>
+            <img [src]="homepage.BUTTON_IMAGE_URL" [alt]="homepage.BUTTON_ALT_TEXT" [matTooltip]="homepage.BUTTON_TOOLTIP_TEXT" />
+        </a>
+    </ng-container>
```

Comment:
If you want to be fresh, use

```suggestion
    @if (config.BRANDING.HOMEPAGE; as homepage) {
        <div class="divider"></div>

        <a [href]="homepage.URL" mat-flat-button color="primary" class="homepage-button">
            <mat-icon>keyboard_return</mat-icon>
            <span></span>
            <img [src]="homepage.BUTTON_IMAGE_URL" [alt]="homepage.BUTTON_ALT_TEXT" [matTooltip]="homepage.BUTTON_TOOLTIP_TEXT" />
        </a>
    }
```